File: Graphics/OpenGL/Startup/app/src/main/java/com/linroid/opengl/MainActivity.java
package com.linroid.opengl;

import android.opengl.GLSurfaceView;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ES20GLSurfaceView surfaceView = (ES20GLSurfaceView) findViewById(R.id.surface_view);
    }
}

File: Graphics/OpenGL/AirHockey/app/src/main/java/com/linroid/airhockey/MainActivity.kt
package com.linroid.airhockey

import android.app.Activity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : Activity() {

    private lateinit var render: AirHockeyRender

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        surfaceView.setEGLContextClientVersion(2)
        render = AirHockeyRender()
        surfaceView.setRenderer(render)
    }

    override fun onResume() {
        super.onResume()
        surfaceView.onResume()
    }

    override fun onPause() {
        super.onPause()
        surfaceView.onPause()
    }
}
